lib/charliecloud.py

```diff
@@ -599,9 +599,10 @@ def exit(code):
 
 def init(cli):
    # logging
-   global log_festoon, log_fp, trace_fatal, verbose
+   global log_festoon, log_fp, trace_fatal, verbose, xattrs
```

Comment:
I bet you haven't worked on any other projects with a variable named `festoon` lol.